import-export-cli/impl/importThrottlePolicy.go

```diff
@@ -0,0 +1,84 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package impl
+
+import (
+	"errors"
+	"fmt"
+	"github.com/go-resty/resty/v2"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+func ImportThrottlingPolicyToEnv(accessOAuthToken, importEnvironment, importThrottlingPolicyFile string, importThrottlePolicyUpdate bool) error {
+	adminEndpoint := utils.GetAdminEndpointOfEnv(importEnvironment, utils.MainConfigFilePath)
+	return ImportThrottlingPolicy(accessOAuthToken, adminEndpoint, importThrottlingPolicyFile, importThrottlePolicyUpdate)
+}
+
+func ImportThrottlingPolicy(accessOAuthToken, adminEndpoint, importPath string, importThrottlePolicyUpdate bool) error {
```

Comment:
We don't need this function, I think. You can embed the implementation of this to the function **importThrottlingPolicy**, and call it directly from **ImportThrottlingPolicyToEnv**, isn't it?